modules/afsmtp/afsmtp.c

```diff
@@ -134,6 +135,7 @@ afsmtp_dd_set_from(LogDriver *d, const gchar *phrase, const gchar *mbox)
 
   g_free(self->mail_from->phrase);
   g_free(self->mail_from->address);
+
   self->mail_from->phrase = afsmtp_wash_string(g_strdup(phrase));
```

Comment:
free-d around line 700. (This still not my code though, only change was the whitespace, which i'll fix)